style.css

```diff
@@ -1 +1,179 @@
 /*  Add your styling here */
+@media(max-width: 1024px){
```

Comment:
media queries should live at the bottom of the stylesheet - CSS is read from top to bottom; this means that any styles defined in this block will now get overwritten by any subsequent styles